batch/batch/worker/worker.py

```diff
@@ -195,7 +204,8 @@ async def init(self):
         with open(f'/etc/netns/{self.network_ns_name}/resolv.conf', 'w') as resolv:
             if self.private:
                 resolv.write('nameserver 169.254.169.254\n')
-                resolv.write('search c.hail-vdc.internal google.internal\n')
+                if CLOUD == 'gcp':
+                    resolv.write('search c.hail-vdc.internal google.internal\n')
```

Comment:
Are we absolutely certain there is no parallel version of this in Azure?

Reply:
Looking at `resolv.conf`, I see what looks like
`<random_id>.bx.internal.cloudapp.net`. We can look into figuring out how that's generated.